docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+##### statistic_max_full_collect_data_size
+
+- Unit: LONG
+- Default: 107374182400
+- Description: The size of the largest partition for automatic collection to collect data. Unit: Byte. If a partition exceeds this value, full collection is discarded and sampled collection is performed instead.
+
+##### statistic_collect_max_row_count_per_query
+
+- Unit: INT
+- Default: 5000000000
+- Description: The maximum number of rows to query for a single analyze task. An analyze task will be split into multiple queries if this value is exceeded.
+
+##### statistic_collect_interval_sec
+
+- Unit: s
+- Default: 300
+- Description: The interval for checking data updates during automatic collection. Unit: seconds.
+
+##### statistic_auto_analyze_start_time
+
+- Unit: STRING
+- Default: 00:00:00
+- Description: The start time of automatic collection. Value range: `00:00:00` - `23:59:59`.
+
+##### statistic_auto_analyze_end_time
+
+- Unit: STRING
+- Default: 23:59:59
+- Description: The end time of automatic collection. Value range: `00:00:00` - `23:59:59`.
+
+##### statistic_sample_collect_rows
+
+- Unit: -
+- Default: 200000
+- Description: The minimum number of rows to collect for sampled collection. If the parameter value exceeds the actual number of rows in your table, full collection is performed.
+
+##### histogram_buckets_size
+
+- Unit: -
+- Default: 64
+- Description: The default bucket number for a histogram.
+
+##### histogram_mcv_size
+
+- Unit: -
+- Default: 100
+- Description: The number of most common values (MCV) for a histogram.
+
+##### histogram_sample_ratio
+
+- Unit: -
+- Default: 0.1
+- Description: The sampling ratio for a histogram.
+
+##### histogram_max_sample_row_count
+
+- Unit: -
+- Default: 10000000
+- Description: The maximum number of rows to collect for a histogram.
+
+##### statistics_manager_sleep_time_sec
+
+- Unit: s
+- Default: 60
+- Description: The interval at which metadata is scheduled. Unit: seconds. The system performs the following operations based on this interval:
+  - Create tables for storing statistics.
+  - Delete statistics that have been deleted.
+  - Delete expired statistics.
+
+##### statistic_update_interval_sec
+
+- Unit: s
+- Default: `24 * 60 * 60`
+- Description: The interval at which the cache of statistical information is updated. Unit: seconds.
+
+##### statistic_analyze_status_keep_second
+
+- Unit: s
+- Default: 259200
+- Description: The duration to retain the history of collection tasks. The default value is 3 days. Unit: seconds.
+
+##### statistic_collect_concurrency
+
+- Unit: -
+- Default: 3
+- Description: The maximum number of manual collection tasks that can run in parallel. The value defaults to 3, which means you can run a maximum of three manual collections tasks in parallel. If the value is exceeded, incoming tasks will be in the PENDING state, waiting to be scheduled.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d0cd373a-e21d-4d3a-8491-854959762f4c)
Wrap the parameter value in code format would be best.